Cython/Utility/Embed.c

```diff
@@ -44,6 +44,8 @@ static int __Pyx_main(int argc, wchar_t **argv)
 
         PyConfig config;
         PyConfig_InitPythonConfig(&config);
+        /* Disable parsing command line arguments */
```

Comment:
We generally exclude comments from the generated C files if they start with `//`. This comment seems worth excluding.
```suggestion
        // Disable parsing command line arguments
```